docs/signer_install.md

```diff
@@ -134,7 +134,7 @@ Please ensure the secret is properly protected by ACLs.
 
 ```shell
 kubectl create secret generic kritis-authority-key \
-  --from-file=public=${PUB_KEY_FILE} --from-file=private=${PRIV_KEY_FILE}
+  --from-file=public=${PUB_KEY_FILE} --from-file=private=${PRIV_KEY_FILE} --from-literal=passphrase=<PASSPHRASE>
```

Comment:
To be consistent with other documentation, use lowercase: <passphrase>

Reply:
I was going for consistency in this specific file, which appears to be using caps for all env vars. WDYT?